<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="mainLayout" stretch="3,1">
     <item>
      <widget class="QWidget" name="mainPlotContainer" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCustomPlot" name="customplot" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="controlPanel" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_SerialConfig">
              <property name="text">
               <string>串口配置</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_SerialPort">
              <property name="text">
               <string>串口号</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="cbPort"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_BaudRate">
              <property name="text">
               <string>波特率</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="cbBaud"/>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="btnSwitchPort">
              <property name="text">
               <string>打开串口</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="btnTest">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="subContainer">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>159</width>
               <height>314</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
